test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/AlternateKeysVocabularyTests.cs

```diff
@@ -50,17 +50,51 @@ public void TestAlternateKeysVocabularyModel()
             Assert.Equal("OData.Community.Keys.V1", alternateKeysTerm.Namespace);
             Assert.Equal("AlternateKeys", alternateKeysTerm.Name);
 
-            StringWriter sw = new StringWriter();
+            StringWriter swAlternateKeys = new StringWriter();
             IEnumerable<EdmError> errors;
-            using (var xw = XmlWriter.Create(sw, new XmlWriterSettings { Indent = true, Encoding = Encoding.UTF8 }))
+            using (var xwAlternateKeys     = XmlWriter.Create(swAlternateKeys,     new XmlWriterSettings { Indent = true, Encoding = Encoding.UTF8 }))
             {
-                Assert.True(model.TryWriteSchema(xw, out errors));
+                Assert.True(model.TryWriteSchema(
+                    nsName => nsName.Equals("OData.Community.Keys.V1") ? xwAlternateKeys : null, 
+                    out errors));
             }
 
-            string output = sw.ToString();
+            Assert.False(errors.Any(), "No Errors");
+
+            string output = swAlternateKeys.ToString();
+            Assert.True(expectedTextAlternateKeys == output, $"expectedTextAlternateKeys not matching");
+        }
+
+        [Fact]
+        public void TestUrlEscapeVocabularyModel()

```

Comment:
Add test cases in:

1. Create Edm function with the "urlEscape" annotation and can write it to CSDL ((inline and outline)
2. Can reader the CSDL with the Edm function decorated with "UrlEscape" annotation, (inline and outline)